src/main/app/src/app/zaken/besluit-view/besluit-view.component.html

```diff
@@ -13,6 +13,9 @@
             <zac-static-text fxFlex="50" [label]="'ingangsdatum' | translate" [value]="besluit.ingangsdatum | datum"></zac-static-text>
             <zac-static-text fxFlex="50" [label]="'vervaldatum' | translate" [value]="besluit.vervaldatum | datum"></zac-static-text>
             <zac-static-text fxFlex="100" [label]="'toelichting' | translate" [value]="besluit.toelichting"></zac-static-text>
+            <div class="table-field-wrapper">
+                <mfb-form-field class="m-1" [field]="besluitInformatieobjecten"></mfb-form-field>
```

Comment:
wat doet de m-1 class?

Reply:
Niets, was overgebleven van eerdere CSS-wijziging. Opgelost in opvolgende commit